static/js/mobile.js

```diff
@@ -0,0 +1,75 @@
+var loadingMessage = document.getElementById('msg');
```

Comment:
__[eslint]__
* Error - 'document' is not defined. (no-undef)

<!-- Comment by [SideCI](https://sideci.com) -->